radanalyticsio/spark-operator/base/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: spark-operator-edit-resources
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+subjects:
+  - kind: ServiceAccount
+    name: spark-operator
+    namespace: default
```

Comment:
is "default" really the intended namespace for this?

Reply:
Kustomize (or kfctl?) will change that to a namespace where this is being submitted - i.e. `opendatahub`, so it does not really matter what value is there, but maybe for the sake of clarity, we could use `opendatahub`

Reply:
Good catch. updated